nodejs/eks/cluster.ts

```diff
@@ -1107,4 +1113,14 @@ export class Cluster extends pulumi.ComponentResource {
             providers: { kubernetes: this.provider },
         });
     }
+    // Generate role-based kubeconfig.
+    getRoleKubeconfig(roleArn: pulumi.Input<aws.ARN>): pulumi.Output<any> {
```

Comment:
Shouldn't this return an `Output<string>` since you're using `stringify`?

Reply:
Good catch. Done.